app/images/curve-low.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="125" height="31" viewBox="0 0 125 31" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Will be useful to optimise the svg: https://jakearchibald.github.io/svgomg/

Reply:
Good call!